upb/json/printer.c

```diff
@@ -208,18 +208,30 @@ static size_t fmt_bool(bool val, char* buf, size_t length) {
   return n;
 }
 
-static size_t fmt_int64(long val, char* buf, size_t length) {
+static size_t fmt_int32(long val, char* buf, size_t length) {
```

Comment:
I think int32 should use `int`.

Reply:
fmt_int32 is also used for int64 for map key